### PR TITLE
Run3-sim69 Add a few cfi files which defines transition energies for standard G4 versions

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Add a few cfi files which defines transition energies for standard G4 versions

#### PR validation:

Use the cfg file in the test area to check this 

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: SimG4Core/PhysicsLists/python/physicsQGSP_BERT_G4104_cfi.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import FWCore.ParameterSet.Config as cms
+from SimG4Core.Configuration.SimG4Core_cff import *
+
+g4SimHits.Physics.EmaxBERT   = cms.double(8.)  # in GeV
```
Comment (civanch) [inline SimG4Core/PhysicsLists/python/physicsQGSP_BERT_G4104_cfi.py:4]: @bsunanda , this 6-8 GeV transition is not justified by our previous testing. Do we really need playing energy transition again? But that we have to go via many combinations, I do not see how we make this choice.

File: SimG4Core/PhysicsLists/python/physicsQGSP_FTFP_BERT_G4104_cfi.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import FWCore.ParameterSet.Config as cms
+from SimG4Core.Configuration.SimG4Core_cff import *
+
+g4SimHits.Physics.EmaxBERT   = cms.double(9.9) # in GeV
```
Comment (civanch) [inline SimG4Core/PhysicsLists/python/physicsQGSP_FTFP_BERT_G4104_cfi.py:4]: This transition is really problematic.

File: SimG4Core/PhysicsLists/python/physicsQGSP_BERT_G4104_cfi.py (hunk 1 above)
Comment (bsunanda) [reply SimG4Core/PhysicsLists/python/physicsQGSP_BERT_G4104_cfi.py:4]: This was the transition region in Geant4.10.4.p03 - I just want the HGCal TB team to see how bad/good it is

File: SimG4Core/PhysicsLists/python/physicsQGSP_FTFP_BERT_G4104_cfi.py (hunk 2 above)
Comment (bsunanda) [reply SimG4Core/PhysicsLists/python/physicsQGSP_FTFP_BERT_G4104_cfi.py:4]: This was again a part of Geant4.10.4.p03

**Comment by civanch** [comment]
+1

**Comment by silviodonato** [comment]
+1

**Pull request merged**